torchtune/training/lr_schedulers.py

```diff
@@ -54,3 +54,27 @@ def lr_lambda(current_step: int) -> float:
         return max(0.0, cosine_lr_multiple)
 
     return LambdaLR(optimizer, lr_lambda, last_epoch)
+
+
+def get_lr(optimizer_in_bwd, optimizer) -> str:
```

Comment:
Based on optimizer_in_bwd, the second input could be pure optimizer or optim_wrapper, do we have a better name for the second input? @weifengpy

Reply:
vanilla_optimizer ?